app/src/main/java/io/apicurio/registry/services/http/RegistryExceptionMapperService.java

```diff
@@ -132,6 +132,7 @@ public class RegistryExceptionMapperService {
         map.put(RoleMappingAlreadyExistsException.class, HTTP_CONFLICT);
         map.put(RoleMappingNotFoundException.class, HTTP_NOT_FOUND);
         map.put(TenantManagerClientException.class, HTTP_INTERNAL_ERROR);
+        map.put(io.apicurio.registry.rest.ConflictException.class, HTTP_CONFLICT);
```

Comment:
Why the fully qualified reference rather than importing?

Reply:
There is already a `ConflictException` in the ccompat API.

Reply:
Ah ok.  Is there a reason to **not** merge them?